app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=in%20the%20database.-,Ask%20for%20confirmation%20of%20email.,-Can%20reset%20password), Please add `:confirmable` module here in line 5 so the app can ask for email confirmation.